tests/test_data/test_pipelines/test_transform/test_transform.py

```diff
@@ -227,6 +221,8 @@ def area(bboxes):
 
     assert (area(results['gt_bboxes']) <= area(gt_bboxes)).all()
     assert (area(results['gt_bboxes_ignore']) <= area(gt_bboxes_ignore)).all()
+    assert results['gt_bboxes'].dtype == np.float32
```

Comment:
gt_labels should also consider asserting.

Reply:
Did not create dummy labels here.

Reply:
I notice that some `gt_labels`'s type is tensor but actually we use np int64 array. I'll change them in this PR.